src/seedu/addressbook/parser/Parser.java

```diff
@@ -73,33 +74,36 @@ public Command parseCommand(String userInput) {
 
         switch (commandWord) {
 
-        case AddCommand.COMMAND_WORD:
-            return prepareAdd(arguments);
+            case AddCommand.COMMAND_WORD:
+                return prepareAdd(arguments);
```

Comment:
You change the indentation rules for switch-case in your IDE config to better follow the repo style guides. 😄

Reply:
I clicked on the auto reformatting option in intellij, and this is the default style. Guess I will change the style to suit this mod's needs :)